SignalServiceKit/src/Messages/Interactions/OWSContact.m

```diff
@@ -490,15 +480,11 @@ + (nullable OWSSignalServiceProtosDataMessageContact *)protoForContact:(OWSConta
     [contactBuilder setNameBuilder:nameBuilder];
 
     for (OWSContactPhoneNumber *phoneNumber in contact.phoneNumbers) {
-        if (!phoneNumber.ows_isValid) {
-            OWSProdLogAndFail(@"%@ phone number is invalid.", self.logTag);
-            continue;
-        }
         OWSSignalServiceProtosDataMessageContactPhoneBuilder *phoneBuilder =
             [OWSSignalServiceProtosDataMessageContactPhoneBuilder new];
         phoneBuilder.value = phoneNumber.phoneNumber;
-        if (phoneBuilder.label.ows_stripped.length > 0) {
-            phoneBuilder.label = phoneBuilder.label.ows_stripped;
+        if (phoneNumber.label.ows_stripped.length > 0) {
```

Comment:
This code isn't tested or even exercised yet and I'm still working out the glitches.